lib/web/fetch/util.js

```diff
@@ -255,16 +255,23 @@ function appendFetchMetadata (httpRequest) {
 
 // https://fetch.spec.whatwg.org/#append-a-request-origin-header
 function appendRequestOriginHeader (request) {
-  // 1. Let serializedOrigin be the result of byte-serializing a request origin with request.
+  // 1. Let serializedOrigin be the result of byte-serializing a request origin
+  //    with request.
+  // TODO: implement "byte-serializing a request origin"
```

Comment:
btw. you could use isomorphicEncode from lib/web/fetch/util.js

Reply:
byte serializing a request origin is much more than isomorphic encoding

Reply:
right